test-api

```diff
@@ -44,4 +43,3 @@ docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose
 docker-compose ps
 
 exit $(docker-compose ps -q | xargs docker inspect -f '{{ .State.ExitCode }}' | grep -v 0 | wc -l | tr -d ' ')
-
```

Comment:
why remove the newline? git always warns me about missing newlines at the end of files